plugins/inputs/openweathermap/README.md

```diff
@@ -0,0 +1,67 @@
+# Telegraf Plugin: openweathermap
+
+OpenWeatherMap provides the current weather and forecasts for more than 200,000 cities. To use this plugin you will need a token. For more information [click here](https://openweathermap.org/appid).
+
+Find city identifiers in this [list](http://bulk.openweathermap.org/sample/city.list.json.gz). You can also use this [url](https://openweathermap.org/find) as an alternative to downloading a file. The ID is in the url of the city: `https://openweathermap.org/city/2643743`
+
+### Configuration:
+
+```toml
+[[inputs.openweathermap]]
+  ## Root url of API to pull stats
+  # base_url = "http://api.openweathermap.org/data/2.5/"
+  ## Your personal user token from openweathermap.org
+  # app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
+  ## List of city identifiers
+  # cities = ["2988507", "519188"]
+  ## HTTP response timeout (default: 5s)
+  # response_timeout = "5s"
+  ## Query the next ten days forecast (default: true)
+  # forecast_enable = true
```

Comment:
I feel like a user would want to grab the forecast on a different interval, maybe every 3 hours, while getting the current weather every 10m.  It could be easier to use separate instances of this plugin at the same time if we could disable the weather lookup, and run one or both options:
```toml
[[inputs.openweathermap]]
  interval = "10m"
  app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
  gather = ["weather"]

[[inputs.openweathermap]]
  interval = "1h"
  app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
  gather = ["5d_forecast"]
```

I'm using an array of items in the example in case we want to add more lookups later.

Reply:
OK good idea!

One tiny issue is that gather seems to conflict (`type OpenWeatherMap has both field and method named Gather`) with the function name, so I'm naming the parameter `fetch`